test/e2e/service.go

```diff
@@ -1368,8 +1370,6 @@ func verifyServeHostnameServiceUp(c *client.Client, ns, host string, expectedPod
 		passed := false
 		pods := []string{}
 		// Retry cmdFunc for upto 5 minutes.
```

Comment:
the comment here suggests the functions would be retried for up to 5 minutes, though the code actually only retries for 1 minute.

in any case, shouldn't we increase the timeout here? or am I misunderstanding this code?